pkg/build/node/systemd.go

```diff
@@ -23,6 +23,7 @@ Description=kubelet: The Kubernetes Node Agent
 Documentation=http://kubernetes.io/docs/
 
 [Service]
+ExecPreStart=/bin/sh -c '[ -e /dev/console ] && [ ! -e /dev/kmsg ] && ln -s /dev/console /dev/kmsg'
```

Comment:
what about doing this in the entrypoint in the base image?

Reply:
we have a pattern for this sort of workaround here: https://github.com/kubernetes-sigs/kind/blob/master/images/base/entrypoint

Reply:
Updated, moved that to the base image's entrypoint. /ptal